packages/vm/test/api/customChain.spec.ts

```diff
@@ -97,13 +96,11 @@ describe('VM initialized with custom state', () => {
     common.setHardfork(Hardfork.London)
     const vm = await createVM({ blockchain, common })
     await vm.stateManager.generateCanonicalGenesis!(genesisState)
-    const sigHash = new Interface(['function retrieve()']).getSighash(
-      'retrieve',
-    ) as PrefixedHexString
+    const calldata = '0x2e64cec1' // This is the call data to trigger the `retrieve` method on the given contract
```

Comment:
```suggestion
    const calldata = new Interface(['function retrieve()']).getFunction('retrieve')!.selector
```

Since we have kept the ethers dependency in I think we should (just like the other example) use the interface to derive the correct selector (for clarity)

Reply:
Let me see if this works without bringing in those node builtins again.  Not sure if Richard got rid of them or not.

Reply:
Whoops, have to import Interface as well :sweat: